File: GameJam/Assets/Scripts/Inventory/Item.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.UI;

public class Item// : MonoBehaviour
{
    public int id;

    public int height = 1;
    public int width = 1;

    private bool rotated = false;

    public Sprite sprite;

    public Item(int id, int ht, int wd, Sprite itemSprite) {
        this.id = id;
        this.height = ht;
        this.width = wd;
        this.rotated = false;
        this.sprite = itemSprite;
    }

    public bool isRotated() {
        return this.rotated;
    }

    public void rotate() {
        int swap = this.height;
        this.height = this.width;
        this.width = swap;

        this.rotated = !this.rotated;
    }
}

File: ProofOfConcept/item.py
class Item:
    height: int
    width: int

    rotated: bool
    id: int

    def __init__(self, id, ht, wd) -> None:
        self.id = id
        self.height = ht
        self.width = wd

    def rotate(self):
        swap = self.height
        self.height = self.width
        self.width = swap